sgp30/sgp30.c

```diff
@@ -696,7 +696,10 @@ int16_t sgp30_iaq_init() {
  *
  * This call aleady initializes the IAQ baselines (sgp30_iaq_init())
  *
- * Return:  STATUS_OK on success, an error code otherwise
+ * Return:  STATUS_OK on success,
+ *          SGP30_ERR_UNKNOWN_FEATURE_SET if the sensor's feature set
+ *                                        is unknown or outdated,
+ *          STATUS_FAIL on any other error
```

Comment:
can we cange to "An error code otherwise" in preparation for future changes with more precise error codes?